aptos-move/framework/aptos-framework/sources/object.move

```diff
@@ -42,6 +42,8 @@ module aptos_framework::object {
     /// The resource is not stored at the specified address.
     const ERESOURCE_DOES_NOT_EXIST: u64 = 7;
 
+    const INIT_GUID_CREATION_NUM: u64 = 0x4000000000000;
```

Comment:
Documentation?

Reply:
Account and Objects can overlap since accounts can be implicitly created. Then someone could send arbitrary coins to the account to create valid event streams that would overlap with the Object event streams. Adding comments.